pytorch_lightning/trainer/trainer.py

```diff
@@ -413,6 +413,10 @@ def __init__(
                     # backprop every 5 steps in a batch
                     trainer = Trainer(truncated_bptt_steps=5)
 
+                Using this feature requires updating your LightningModule's `training_step()` to include
+                a `hiddens` arg.
+
```

Comment:
pls remove blank lines here and the next continue line shall start with extra space